kernelci/runtime/lava.py

```diff
@@ -66,10 +66,10 @@ class Callback:
 
     # LAVA job result names
     LAVA_JOB_RESULT_NAMES = {
-        COMPLETE: "PASS",
-        INCOMPLETE: "FAIL",
-        CANCELED: "UNKNOWN",
-        CANCELING: "UNKNOWN",
+        COMPLETE: "pass",
+        INCOMPLETE: "incomplete",
+        CANCELED: "incomplete",
+        CANCELING: "incomplete",
```

Comment:
What status does lava use for failed jobs?

Reply:
I mean, how would we denote that the job ran successfully but somehow failed?

Reply:
Unfortunately LAVA documentation seems not very consitent on this topic. For example here is described LAVA TestJob state machine:
https://lava.readthedocs.io/en/latest/technical-references/state-machine/#testjobs
But there is nothing about results, it has paragraph "Results" in that doc, but it is empty. 
So what i did, i submitted job with invalid rootfs URL, recorded callback details/content, and noted which result code job will return on invalid rootfs url (that will cause job to fail shortly after submission), and what result code returned on "good" job.

Reply:
Yes, it's weird that the documentation is empty.
So what result code did you receive?
Maybe @pawiecz is aware of the different lava job results?